_positions/TAM-cloud.gov.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Technical Account Manager — Cloud.gov 
+permalink: roles-and-teams/positions/technical-account-manager-cloud-dot-gov/
+team: Infrastructure
+active: true
+---
+
+###Role summary
+
+As a technical account manager for the Cloud.gov, you’ll act as a product ambassador to our partners, representing products to to them and ensuring they can easily use them. Additionally, you’ll ensure that our partners maintain a strong relationship with the 18F team. The 18F team and its partners will rely on you to assist with product understanding, documentation, and increasing business. 
+
+This position is actually a hybrid between a technical account manager and a business account manager. We’re looking for someone who can talk tech and business with clients, the 18F team, and stakeholders — someone who has experience in government or highly regulated industries (such as financial markets, health, and so on). This position is best suited to someone with considerable experience in documentation, sales, and product management.
+
+
+###Key objectives
+
+**Objective \#1: Be a strong ambassador of the product.**
+- Have a full understanding of the product; deftly answer inquiries about it and describe how the product will exceed the partner’s  needs.
+- Determine whether a given product is right for the partner; help them understand why there is (or isn’t) a fit. 
+- Always provide a clear value proposition explanation to make a sale. 
+- Demonstrate technical insight: Understand the technical capabilities of the product to be able to answer questions on what the product can and cannot do. 
+
+
+
+**Objective \#2: Practice an exceptional level of customer service with all partners, providing a unique, tailored experience.**
+— Explain the product to people who have varying levels of technical knowledge — always meet the client where they’re at. 
```

Comment:
@hestherchang Looks like you committed a change to correct this hyphen but it might still be an issue.